code/datums/loadout/loadout_helper.dm

```diff
@@ -174,3 +174,12 @@
 		return
 	var/datum/loadout/loadout = jatum_deserialize(loadout_json)
 	return loadout
+
+/proc/convert_loadouts_list(list/loadouts_data)
+	var/list/new_loadouts_data = list()
+	for(var/i = 1; i <= length(loadouts_data); i += 2)
+		var/next_loadout_data = list()
```

Comment:
for(var/i=0 to 88 step 2)

Reply:
i've tried step before, it's a pain in the ass. This just doesn't work, while thise works perfectly.

And this syntax is perfectly clear, it's the standard for all major language

Reply:
It is but not for byond